### ISSUE TITLE
Grouped Segments should use Group Segment Variables

### ISSUE THREAD

**Issue by davidpanzarella** [Semantic-Org/Semantic-UI-LESS#49]
Shouldn't this use `@groupedBoxShadow`? Seems this variable isn't applied to any Group Segments besides the standalone version.

https://github.com/Semantic-Org/Semantic-UI-LESS/blob/35ae43acde6d65938317a7e3a3a577ac7ff0c206/definitions/elements/segment.less#L356

**Comment by davidpanzarella** [comment]
Crickets in here it seems ...

**Comment by davidpanzarella** [comment]
And.... this is why we'll moving from SUI

**Comment by nullbio** [comment]
@davidpanzarella Just got hit by a bug that's over 3 years old (#6). Seems this project is dead and now I'm second guessing my decision. Do you know of any other frameworks that are similar to this but still active?

**Comment by davidpanzarella** [comment]
@nullbio Yeah, we're moving away from SUI altogether. Far too many bugs, and no good support. We're either moving to https://atlaskit.atlassian.com/, https://pro.ant.design/, or something else. This is a decent list here as well: https://hackernoon.com/23-best-react-ui-component-libraries-and-frameworks-250a81b2ac42

Evergreen is also a new kid on the block that is looking promising: https://evergreen.segment.com

**Comment by lubber-de** [comment]
Fixed for the active community fork Fomantic-UI by https://github.com/fomantic/Fomantic-UI/pull/2186
